### PR TITLE
Adding py3.13 to tests

### PR DESCRIPTION
**Pull request by robotrapta**
Adding python 3.13 to the test matrix.  Updating old references to versions of python <3.9 which we don't support any more.

### CODE REVIEW

**Comment by tyler-romero** [comment]
Can you wait for https://github.com/groundlight/python-sdk/pull/279 ? This includes all the updates to 3.8 -> 3.9

**Comment by robotrapta** [comment]
> Can you wait for https://github.com/groundlight/python-sdk/pull/279 ? This includes all the updates to 3.8 -> 3.9

Sure thing.  I can rebase on top of it even.

File: generated/setup.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
     author_email="support@groundlight.ai",
     url="",
     keywords=["OpenAPI", "OpenAPI-Generator", "Groundlight API"],
-    python_requires=">=3.6",
+    python_requires=">=3.9",
```
Comment (tyler-romero) [inline generated/setup.py:35]: I think this is the only change I didnt capture - although this file is generated so I'm not sure if its the right place to make the change.

Comment (robotrapta) [reply generated/setup.py:35]: Yeah, this file AFAIK isn't actually used by anything.

**Comment by robotrapta** [comment]
Tyler beat me to it!

**Pull request closed without merging**